mocks/handlers.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Would preferably not have this disabled here, what is the reason?

Reply:
resolved by typing the label